general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto.md

```diff
@@ -0,0 +1,28 @@
+---
+title: How to connect an xDai account to MyCrypto
+tags:
+  - xDai
+date_published: '2021-03-17'
+date_modified: '2021-03-17'
+---
+
+If you want one place to view and manage all of your xDai accounts at once, you’ll want to get set up with MyCrypto.
+
+1. Visit [app.mycrypto.com/add-account](https://app.mycrypto.com/add-account/)
+2. Select the wallet you’ll be connecting to MyCrypto. _Note: click [View Address](/how-to/accessing-wallet/how-does-view-address-work) if you only plan to track an account and not send from it._
+3. Select the xDai network from the dropdown. _Note: If you’re using MetaMask, you will not see this dropdown and you’ll need to make sure you’ve selected the xDai network within MetaMask before continuing._
+
+![Select network](../../assets/general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto/select-network.png)
+
+4. Continue to connect, and check out your xDai account(s) on your MyCrypto dashboard, right alongside any other accounts you have!
+
+![Accounts overview](../../assets/general-knowledge/scaling/how-to-connect-an-xdai-account-to-mycrypto/accounts-overview.png)
+
+As mentioned earlier, you can use the same account for Ethereum and for xDai, so we’ve made sure to distinguish your accounts separately according to the network they’re attached to. Note the above screenshot where it shows two separate listings for one address — one used with Ethereum, one used with xDai.
```

Comment:
"As mentioned earlier" > "As mentioned [in this article](url)" and please link to the appropriate article where it's mentioned. I think it's the 'what is xdai' article

Reply:
nope jk its referenced in the the 'how to get xdai article'

Reply:
Thanks!